test/unit/math/mix/fun/arg_test.cpp

```diff
@@ -0,0 +1,15 @@
+#include <test/unit/math/test_ad.hpp>
+#include <complex>
+#include <vector>
+
+TEST(mixScalFun, arg) {
+  auto f = [](const auto& x) { return arg(x); };
+
+  // undefined with 0 in denominator
+  stan::test::expect_ad(f, std::complex<double>(0.9, 0.8));
+  for (double im : std::vector<double>{-1.3, 2.3}) {
```

Comment:
I think this should work for any of the common complex args since it depends on atan2 and not atan.

Reply:
There's a divide by zero if the real part is zero.  So all the finite diff tests fail.